reference/6/Microsoft.PowerShell.Utility/Invoke-RestMethod.md

```diff
@@ -16,9 +16,9 @@ Sends an HTTP or HTTPS request to a RESTful web service.
 ## Syntax
 
 ```powershell
-Invoke-RestMethod [-Method <WebRequestMethod>] [-UseBasicParsing] [-Uri] <Uri>
+Invoke-RestMethod [-Method <WebRequestMethod>] [-CustomMethod <String>] [-UseBasicParsing] [-Uri] <Uri>
  [-WebSession <WebRequestSession>] [-SessionVariable <String>] [-Credential <PSCredential>]
- [-UseDefaultCredentials] [-CertificateThumbprint <String>] [-Certificate <X509Certificate>]
+ [-UseDefaultCredentials] [-CertificateThumbprint <String>] [-Certificate <X509Certificate>] [-SkipCertificateCheck]
```

Comment:
Can you add `-FollowRelLink` and `-MaximumFollowRelLink <int>` here?

Reply:
Done.